test/unit/error.test.ts

```diff
@@ -546,6 +548,56 @@ describe('MongoErrors', () => {
       expect(isResumableError(new MongoNetworkError('ah!'))).to.be.true;
     });
 
+    it('should return false for Error with code', () => {
+      const errorWithCode = new (class extends Error {
+        code = MONGODB_ERROR_CODES.CursorNotFound;
+      })();
+      expect(isResumableError(errorWithCode)).to.be.false;
+    });
+
+    it('should return false nullish arguments', () => {
```

Comment:
```suggestion
    it('should return false for nullish error argument regardless of wire version', () => {
```
and the assertion in line 561 should pass in and 8 and a 9 instead of null for the second argument as well as test undefined for the error arg